tb_plugin/torch_tb_profiler/profiler/module_op.py

```diff
@@ -153,13 +182,16 @@ def _aggregate_modules(modules: Iterable[ModuleNode]) -> Dict[Tuple[str, int], M
     return module_aggs
 
 
-def _get_module_list(tid2tree: Dict[int, OperatorNode]) -> Generator[ModuleNode, None, None]:
+def _get_module_list(tid2tree: Dict[int, OperatorNode], get_pl_module = False) -> Generator[OperatorNode, None, None]:
```

Comment:
better to pass the class instead of bool flag?

just like: _get_module_list(tid2tree, ModuleNode) or _get_module_list(pl_tid2tree, PLModuleNode)

Reply:
In this way, the logic of _get_module_list will be much simpler:

```
        if isinstance(node, module_cls)
            yield node
```
instead of 
```
        if isinstance(node, ModuleNode) and not get_pl_module:
            yield node

        if isinstance(node, PLModuleNode) and get_pl_module:
            yield node

```

Reply:
thanks. done